README.md

```diff
@@ -79,6 +79,7 @@ Consider submitting to [the deno.land/x](https://github.com/denoland/deno_websit
 - [dinatra](https://github.com/syumai/dinatra) - Sinatra like light weight web app framework for deno.
 - [djwt](https://github.com/timonson/djwt) - Make JSON Web Tokens (JWT) on deno based on JWT and JWS specifications.
 - [dso](https://github.com/manyuanrong/dso) - A simple ORM library based on mysql.
+- [evt](https://evt.land) - Type safe replacement for EventEmitter.
```

Comment:
I'd prefer to target directly the github repo ( https://github.com/garronej/evt ) instead of the website. Your thought @kevinkassimo  ?

Would be more consistent with the rest of the exposed modules.

Reply:
Ok, no problem I understand. 👍